### PR TITLE
controller for project. Add fields to projects table

### PR DESCRIPTION
**Pull request by yuriy-ivanchenko**
Stil cannot choose a mentor for a project

### CODE REVIEW

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -1,7 +1,60 @@
-# frozen_string_literal: true
```
Comment (andrzejkrzywda) [inline app/controllers/projects_controller.rb]: Good change, I don't like those comments everywhere.

File: app/helpers/projects_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module ProjectsHelper
```
Comment (andrzejkrzywda) [inline app/helpers/projects_helper.rb]: Can you remove such empty files?

File: app/views/projects/_project.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1,2 @@
+json.extract! project, :id, :created_at, :updated_at
```
Comment (andrzejkrzywda) [inline app/views/projects/_project.json.jbuilder]: We probably don't need this file, right?

File: app/views/projects/index.json.jbuilder [hunk 4]
```diff
@@ -0,0 +1 @@
+json.array! @projects, partial: "projects/project", as: :project
```
Comment (andrzejkrzywda) [inline app/views/projects/index.json.jbuilder]: Do we need that?

File: app/views/projects/show.json.jbuilder [hunk 5]
```diff
@@ -0,0 +1 @@
+json.partial! "projects/project", project: @project
```
Comment (andrzejkrzywda) [inline app/views/projects/show.json.jbuilder]: to be removed

**Comment by andrzejkrzywda** [comment]
Thank you, I'm merging this.

Could you please create a new PR where you remove the not needed files? I prefer to keep the repo clean without unused stuff.

**Comment by oxanayoxana** [comment]
Hi @yuriy-ivanchenko!
Thanks for the controller and form!

Regarding your https://github.com/arkency/find-open-source-mentor/pull/19#issue-314870236
I think you can add a mentor via `hidden field` in the _form partial. Currently we don't have separate roles for mentor and for developer, so you can just pass the `current user` to this hidden field. Something like that: 
`<%= form.hidden_field :user_id, as: :hidden, input_html: { value: current_user.id } %>` should work I believe.

Also, I think we should change the type of project's description - now it is `string`, but maybe type `text` will be better, so that we could add another field to this _form partial to provide description (apart from name).

Also, I'd suggest to change `<%= link_to 'New Project', new_project_path %>` in the project's index view to `<%= link_to 'Add New Project', new_repository_path, method: :get %>`, so that once current_user clicks on  'Add New Project' button, he/she is redirected to the list of one's github repositories, and is able to chose the one to add as a project. 

/// Sidenote (below is just my thinking - not concrete steps for you to do, so)
Once he/she clicks "Add this repo", page redirects him to `new_project_path` and mainly - your _form partial, where he adds name of the project, description etc. I've already changed button's path to `new_project_path` in last commit (), so you don't need to do anything here. The only thing to figure out is how to pass this chosen repo to the project's _form as a hidden_field and store it as a `link` param in project's table... I don't know how to do that yet, if you have ideas, feel free to share them!

**Comment by yuriy-ivanchenko** [comment]
@oxanayoxana got you

**Comment by yuriy-ivanchenko** [comment]
@andrzejkrzywda @oxanayoxana have a suggestion to create a seed file because it will allow devs more comfortable way of developing on the local machine

**Comment by oxanayoxana** [comment]
@yuriy-ivanchenko I never used seed files, if you can create it - feel free to do that ☺️

**Pull request merged**